ipstack/icon_ipstack/actions/lookup/schema.py

```diff
@@ -110,7 +110,7 @@ class LookupOutput(komand.Output):
     },
     "time_zone": {
       "type": "string",
-      "title": "Public IP Address in Decimal",
```

Comment:
This title is correct

Reply:
Validators fail on it.

Reply:
Specifically, the lowercase i in the word IN

Reply:
reverted